### DISCUSSION TITLE
Having a chat

### DISCUSSION THREAD

**Discussion by waghanza** [the-benchmarker/web-frameworks#3790] [Show and tell]
Hello @the-benchmarker/web-frameworks,

I have create a `slack` channel => https://join.slack.com/t/thebenchmarker/shared_invite/zt-fcyy1ybq-A7T1SedewiVMEtJQGEyQYw

The idea of this chat is that every automated stuff could create notification on it. For example, if any framework fails, a slack notification could be sent on slack.

I prefer to use GitHub discussions for new ideas and other.

Regards,

**Comment by dalisoft** [comment]
@waghanza I prefer Telegram for chat, very easy and nice UI (maybe for me only). After that, Github Discussion (very nice way too)
